docs/metrics.md

```diff
@@ -88,6 +94,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
 - `cd5`
 - `cd6`
 - `cd10`
+- `cd11`
+
+#### `new-note`
+Event fired when a user creates a new note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `list-view` where `in-note` refers to the meatball menu flow and `list-view` refers to the button in the list view
```

Comment:
A third option to create a new note is using `send-to-note` while being on the `list-view`. I will add that 👍